derive/Cargo.toml

```diff
@@ -2,7 +2,9 @@
 name = "clickhouse-derive"
 version = "0.2.0"
 description = "A macro for deriving clickhouse::Row"
-authors = ["Paul Loyd <pavelko95@gmail.com>"]
+authors = ["ClickHouse Contributors <???>"] # REVIEW
```

Comment:
@slvrtrn hm?

Reply:
+1 to keep the original author

Reply:
This field is more like "maintainers". So, I prefer to keep here relevant email